circe/src/main/scala/org/http4s/circe/CirceInstances.scala

```diff
@@ -92,20 +92,52 @@ trait CirceInstances extends JawnInstances {
   implicit def jsonEncoder[F[_]: Applicative]: EntityEncoder[F, Json] =
     jsonEncoderWithPrinter(defaultPrinter)
 
-  def jsonEncoderWithPrinter[F[_]: Applicative](printer: Printer): EntityEncoder[F, Json] =
+  private def fromJsonToChunk(printer: Printer, cs: Charset)(json: Json): Chunk[Byte] =
```

Comment:
JSON _must_ be encoded as UTF-8, UTF-16, or UTF-32.  And anything but UTF-8 is generally a bad idea: https://tools.ietf.org/html/rfc7159#section-8.1.  I'm not sure we want to parameterize this, and if we do, perhaps as a sealed type that permits only one of the three JSON encodings.

Reply:
Didn't know about that, I guess the best would be to just remove that change and leave the default.